projects/cashmere/src/lib/sass/_mixins.scss

```diff
@@ -48,6 +48,6 @@
 }
 @mixin formFieldFocusedState() {
     box-shadow: 0 0 5px $blue-glow;
-    border: 1px solid $blue-glow;
+    //border: 1px solid $blue-glow;
```

Comment:
Please delete this line of code instead of leaving it commented out.

Reply:
Sorry about that! I've removed this.